src/sidebar/components/test/group-list-test.js

```diff
@@ -369,20 +373,7 @@ describe('groupList', function() {
       ];
 
       // Configure only one group.
-      const groups = [
-        {
-          id: 'h-devs',
-          links: {
-            html: PRIVATE_GROUP_LINK,
-          },
-          name: 'Hypothesis Developers',
-          organization: groupFixtures.defaultOrganization(),
-          type: 'private',
-        },
-      ];
-      fakeGroups.all = () => {
-        return groups;
-      };
+      groups = [privateGroup];
```

Comment:
Just wanna point out that by reusing `privateGroup` here, you are reusing an object reference instead of creating a new, separate object as the old code did...in case there could be any unexpected side effects from this...

Reply:
hmmm...yeah there shouldn't be because the store shouldn't be modifying the data but I suppose if we wanted to be safer we could move these inside the before each.